Automattic/vip
# Update the `search-replace` command to follow the VIP-CLI style guide

## Description

Updated the voice, tense, and format of command descriptions, examples, and usage models for the `vip search-replace` command to follow the VIP-CLI style guide.

## Pull request checklist

- [ n/a ] Update [SETUP.md](https://github.com/Automattic/vip-cli/blob/trunk/docs/SETUP.md#list-of-environmental-variables) with any new environmental variables.
- [ x ] Follow the [pull request checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#new-pull-requests)
- [ x ] Add/update [automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) as needed.

## New release checklist


## Steps to Test

Observe the results of the `--help` display for the impacted commands:

- `vip`
- `vip search-replace`
